tsconfig.json

```diff
@@ -1,20 +1,27 @@
 {
   "compilerOptions": {
-    "experimentalDecorators": true,
+    "composite": true,
+    "emitDeclarationOnly": true,
+    "declarationDir": "./build/types",
+
     "target": "ES2022",
     "module": "ES2022",
+    "lib": ["dom", "es2022"],
+
+    "jsx": "react",
     "moduleResolution": "Node",
     "skipLibCheck": true,
+
     "allowSyntheticDefaultImports": true,
+    "experimentalDecorators": true,
+    "noImplicitOverride": true,
     "resolveJsonModule": true,
-    "noEmit": true,
-    "jsx": "react",
     "useDefineForClassFields": true,
-    "noImplicitOverride": true,
+
     "paths": {
       "@xh/hoist/*": ["./*"]
-    },
-    "lib": ["dom", "es2022"]
+    }
   },
-  "exclude": ["node_modules"]
+  "include": ["./**/*", "package.json"],
+  "exclude": ["node_modules", "build"]
```

Comment:
New `include` now required, as with `composite:true` you need an include directive (or an even more explicit list of files).  Needed to add `package.json` due to import in EnvironmentService

Expanded `exclude` to include `build` as otherwise generated declarations from one run are attempted to be compiled by the next.  Build outputs clearly shouldn't be inputs.